final/conclusion.tex

```diff
@@ -1,6 +1,35 @@
 \chapter{Conclusion}
 \label{cha:conclusion}
 
+Over the course of the last quarter the Spoofax Language Workbench has been
+explored and a read-eval-print-loop has been created that operates with any
+language defined in the Spoofax Language Workbench.
+
+\Cref{cha:introduction} gave the required background knowledge needed to
+understand the problem domain. \Cref{cha:probl-defin-analys} framed the problem
+definition in the context of this background. \Cref{cha:design} and
+\cref{cha:implementation} discussed the design and the implementation of the
+final product. \Cref{cha:evaluation} evaluated both the final product and the
+process by which it came to be. \Cref{cha:discussion} reflected upon the
+project, after which \cref{cha:recommendations} made recommendations to improve
+the final product.
+
+To successfully complete the project, extensive knowledge needed to be gained of both the conceptual
+ideas behind programming language implementations and of the Spoofax API
+implementing these concepts. This resulted in a slower pace than anticipated.
+
+Once the required knowledge was attained, worthwhile features have been
+contributed upstream. First and foremost, a properly functioning REPL,
+comparable in features to those of popular programming languages. Secondly,
+significant changes have been contributed to DynSem.
+
+Despite having faced significant challenges during the start of the project,
+the most important goals as set forth in the problem description have been
+achieved. Therefore, the project team is still quite satisfied with the end
+result: while not all requested features have been implemented, it has been shown
+that it is possible to create a functioning REPL for any language defined in
+Spoofax, requiring only minimal changes to the language definition.
```

Comment:
"minimal configurations" maybe, otherwise it may look like as if one has to change the definition of the language (i.e. has to change semantics just to have it work with the REPL)